### PR TITLE
Practitioner 

### PR DESCRIPTION
**Pull request by Kolbein**
Not quite sure if the identifier inside qualification should be Organization as well.

Works to create, update and remove Practitioner to/from database.

### CODE REVIEW

File: src/models/practitioner.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+		'gender',
+		'birthDate',
+		'photo',
+		'qualification',
+		'communication',
+		'organization'
+	];
+
+	Schema.statics.getAll = function (args) {
+		return new Promise(async (resolve, reject) => {
```
[the remark this answers is not in the document]
Comment (sureshHARDIYA) [reply src/models/practitioner.js]: @hunghkit  for now, we agreed to keep the promise like the example. Once we update, once resource, I will optimize other files.

**Comment by sureshHARDIYA** [comment]
@Kolbein  please update the PR,

-  Update from master branch
-  Update and move the readme examples to WIKI, as it is getting bigger.

File: src/models/practitioner.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kolbein) [reply src/models/practitioner.js]: Sorry, didn't see your comment, @sureshHARDIYA. Just finished async-await changes for Observation and Practitioner.

**Comment by sureshHARDIYA** [comment]
@Kolbein  Please update this PR, from the master branch. Second, resolve the conflict with readme file and push the changes. Then I can merge it into master.

**Pull request merged**